.github/workflows/nextjs.yml

```diff
@@ -0,0 +1,93 @@
+# Sample workflow for building and deploying a Next.js site to GitHub Pages
```

Comment:
This workflow is great.

In the future I think we should add the following
1) Run Test
2) Run Lint

Either in a separate workflow or in this workflow.

Reply:
I have recently added the Run Lint to the nextjs.yml. Just do next lint.

Reply:
I wanted to add lint and tests to the github flow.

It guarantees any code we deploy meets our minimum quality requirements.

You don't have to do that now.